client/signup/steps/site-style/index.jsx

```diff
@@ -153,10 +152,13 @@ const mapDispatchToProps = ( dispatch, ownProps ) => ( {
 } );
 
 export default connect(
-	state => ( {
-		siteStyle: getSiteStyle( state ),
-		siteType: getSiteType( state ),
-		styleOptions: getSiteStyleOptions( getSiteVerticalName( state ) ),
-	} ),
+	state => {
+		const siteType = getSiteType( state );
+		return {
+			siteStyle: getSiteStyle( state ),
+			siteType,
```

Comment:
Looks like `siteType` is not used anywhere in the component, so let's remove it.